README.adoc

```diff
@@ -47,7 +47,7 @@ using, and developing Minishift:
 
 - https://docs.openshift.org/latest/minishift/getting-started/index.html[Getting started with Minishift]
 - https://docs.openshift.org/latest/minishift/using/index.html[Using Minishift]
-- https://docs.openshift.org/latest/minishift/developing/index.html[Developing Minishift]
+- https://docs.openshift.org/latest/minishift/contributing/developing.html[Developing Minishift]
```

Comment:
+1 for renaming the link as well. Also this link appears twice in the README. Let's fix both links and rename then and we are good to merge.